tests/test_sanic.py

```diff
@@ -0,0 +1,41 @@
+from sanic import Sanic
+from sanic.response import HTTPResponse
+from sanic.utils import HOST, PORT
+from unittest.mock import MagicMock
+import pytest
+import asyncio
+
+
+async def stop(app):
+    await asyncio.sleep(0.2)
```

Comment:
Can we make this a shorter interval, or is sleep required for the demonstration? If we add too many sleep intervals to the tests they're going to start taking forever to run :) So, I think it makes sense to only use them when needed.